lib/ddtrace/analytics.rb

```diff
@@ -8,6 +8,12 @@ def set_sample_rate(span, sample_rate)
         return if span.nil? || !sample_rate.is_a?(Numeric)
         span.set_metric(Datadog::Ext::Analytics::TAG_SAMPLE_RATE, sample_rate)
       end
+
+      def set_measured(span, value = true)
+        return if span.nil?
+        value = [true, 1].include?(value) ? 1 : 0
```

Comment:
We likely want to extract `[true, 1]` into a constant, so that a new array is not allocated on each method call.

Reply:
Yeah, I thought exactly the same thing. Originally I had it in two `==` clauses instead, but Rubocop complained. I think I'm going to revert this back to that original clause and just disable the Rubocop rule, as the constant feels overkill.